modules/ROOT/pages/java-se.adoc

```diff
@@ -23,5 +23,5 @@ For more information, see https://openliberty.io/blog/2019/02/06/java-11.html[Op
 
 Due to differences between Java SE 8 and Java SE 11, an Open Liberty application that runs on Java SE 8 might not run on Java SE 11. For more information, see https://docs.oracle.com/en/java/javase/11/migrate/index.html#JSMIG-GUID-C25E2B1D-6C24-4403-8540-CFEA875B994A[Oracle Java SE 11 migration guide].
 
-== Java SE 15
-Open Liberty runs on any recent Java SE 15 release. Java SE 15 is not a long-term supported release and standard support is scheduled to end in March 2021. Keep in mind, if you download your Java SDK from https://adoptopenjdk.net/index.html?variant=openjdk16&jvmVariant=openj9[AdoptOpenJDK], https://www.eclipse.org/openj9/[Eclipse OpenJ9] has a better memory footprint and startup profile than https://openjdk.java.net/groups/hotspot/[HotSpot]. For more information, see Open Liberty and Java 11.
+== Java SE 17
+Open Liberty is currently supported on the Hotspot OpenJDK version of Java SE 17. Java SE 17 is a long-term supported release and can be downloaded from [jdk.java.net](https://jdk.java.net/17/).  Due to differences between Java SE 11 and Java SE 17, an Open Liberty application that runs on Java SE 11 might not run on Java SE 17. For more information, see [Oracle Java SE 17 migration blog](https://blogs.oracle.com/javamagazine/migrate-to-java-17).
```

Comment:
Are we not pointing to Semeru for Java 17 download? I only ask because we added a link for it to the Java 8 and 11 sections. Also, those sections advise that OpenJ9 has a a better startup profile than HotSpot, but for Java 17, HotSpot appears to be the only option. Just want to check that's how it should be, and if so, do the prior recommendations for Open J9 over HotSpot still stand?